docs/src/main/tut/docs/akkahttp.md

```diff
@@ -3,3 +3,102 @@ layout: docs
 title: Akka - Http
 permalink: /docs/integrations/akkahttp/
 ---
+
+#### Akka Http
+
+[Akka HTTP](http://doc.akka.io/docs/akka-http/10.0.5/java/http/introduction.html) (formerly known as `spray`) is an Akka-based library for implementing HTTP services.
+Akka HTTP is frequently used to write server-side (a.k.a back-end) REST APIs; for instance, we (47 Degrees) use it to write the back-ends
+for the [9Cards](https://github.com/47deg/nine-cards-backend) or the [ScalaDays](http://scaladays.org/) mobile applications.
+Akka HTTP makes writing such APIs easy, thanks to its [routing DSL](http://doc.akka.io/docs/akka-http/10.0.5/java/http/introduction.html).
+In this DSL, routes are written using directives, that filter incoming requests, and end in a `complete` directive.
+The `complete` directive can take just a message, or just the status code to give back; but more frequently, it takes as an argument
+an expression, or program call, whose result is the _response body_ that is given back to the client.
+
+The goal of `freestyle` integration with Akka HTTP is to allow programmers to pass an expression of type `fs : FreeS[F, A]`,
+for some kind-1 type `F[_]` and some base type `A`, as the argument of a `complete` directive. With respect to this goal,
```

Comment:
"some kind-1 type `F[_]`" may be a bit unfriendly for beginners.

Do we need to explain what `F` and `A` are?